### ISSUE TITLE
Libraries version used 

### ISSUE THREAD

**Issue by xElfman** [pruthvi03/DeepSortWithYolov3#2]
Hi, I come across your code from YouTube and I am currently working on your source code. I know that your code referred to emasterclassacademy, which is from here: https://github.com/emasterclassacademy/Single-Multiple-Custom-Object-Detection-and-Tracking

However, I have some problems with the source you used especially the libraries version. Did you manage to follow all the libraries version used in emasterclassacademy?

Thanks.

**Comment by pruthvi03** [comment]
Yes, @xElfman, I referred emasterclassclassacademy but I don't think there was an issue with the library version.
